web-ng/src/ground-maps/ground-maps.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ground-maps',
+  templateUrl: './ground-maps.ng.html',
+  styleUrls: ['./ground-maps.css'],
+})
+export class GroundMaps {
```

Comment:
Nit: I assume instances of this class represent a single map? If so it should be named `GroundMap`.

Also, what's the convention on prefixing app-specific components in Angular? Should we choose a short prefix (gnd-*) and use it for app components?

Reply:
I think `GroundMap` make sense, I think I was thinking Google Maps and put `Maps` there. In terms of `ground-*` or `gnd-*`, I've only seen guides saying "don't abbreviate key words like 'module', 'service', etc.", I am okay with both, I can change them to `gnd` and let's stick to this.